pkg/scheduler/metrics/metrics.go

```diff
@@ -97,13 +97,20 @@ var (
 			StabilityLevel: metrics.STABLE,
 		}, []string{"queue"})
 	SchedulerGoroutines = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Subsystem:         SchedulerSubsystem,
+			DeprecatedVersion: "1.26.0",
+			Name:              "scheduler_goroutines",
+			Help:              "Number of running goroutines split by the work they do such as binding. This metric is replaced by the \"goroutines\" metric.",
+			StabilityLevel:    metrics.ALPHA,
```

Comment:
In theory, we can. But in practice, we would probably be breaking some users too suddenly. One release is not too bad for us to take the burden of maintenance.

Reply:
Yeah, `scheduler_goroutines` was introduced in v1.17 and it's expected that many users are already using it.
https://github.com/kubernetes/kubernetes/pull/83535
It's kind to announce it's deprecated and wait some time so that users can migrate to the new metric.